pkg/runner/logger.go

```diff
@@ -78,6 +81,12 @@ func (f *stepLogFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 				entry.Message = strings.ReplaceAll(entry.Message, v, "***")
 			}
 		}
+
+		for _, v := range *f.masks {
+			if v != "" {
+				entry.Message = strings.ReplaceAll(entry.Message, v, "***")
```

Comment:
Since we are using the `***` for insecure secrets maybe, we want to consider to change the output for masked secrets? Just thinking out loud, definitely not a show stopper for me.

Reply:
This is currently en-par with what GitHub does, so we should keep it.

Reply:
@Poltergeist "insecure secrets" (as in the act CLI flag) means that act will print all secrets and not mask them:
```
 --insecure-secrets                 NOT RECOMMENDED! Doesn't hide secrets while printing logs.
```